src/content/developers/tutorials/how-to-mint-an-nft.md

```diff
@@ -0,0 +1,275 @@
+> Double check that the hashcode you copied links to your **metadata.json** by loading [https://gateway.pinata.cloud/ipfs/<metadata-hash-code>](https://gateway.pinata.cloud/ipfs/%3Chash-code%3E) into a separate window. The page should look similar to the screenshot below:
+
+![Your page should display the json metadata](https://cdn-images-1.medium.com/max/5200/0*I52WF3hN76kPX4An)*Your page should display the json metadata*
+
+Altogether, your code should look something like this:
+
+    require('dotenv').config();
+    const API_URL = process.env.API_URL;
+    const PUBLIC_KEY = process.env.PUBLIC_KEY;
+    const PRIVATE_KEY = process.env.PRIVATE_KEY;
+
+    const { createAlchemyWeb3 } = require("@alch/alchemy-web3");
+    const web3 = createAlchemyWeb3(API_URL);
+
+    const contract = require("../artifacts/contracts/MyNFT.sol/MyNFT.json");
+    const contractAddress = "0x81c587EB0fE773404c42c1d2666b5f557C470eED";
+    const nftContract = new web3.eth.Contract(contract.abi, contractAddress);
+
+    async function mintNFT(tokenURI) {
+      const nonce = await web3.eth.getTransactionCount(PUBLIC_KEY, 'latest'); //get latest nonce
+
+    //the transaction
+      const tx = {
+        'from': PUBLIC_KEY,
+        'to': contractAddress,
+        'nonce': nonce,
+        'gas': 500000,
+        'data': nftContract.methods.mintNFT(PUBLIC_KEY, tokenURI).encodeABI()
+      };
+
+    const signPromise = web3.eth.accounts.signTransaction(tx, PRIVATE_KEY);
+      signPromise.then((signedTx) => {
+
+    web3.eth.sendSignedTransaction(signedTx.rawTransaction, function(err, hash) {
+          if (!err) {
+            console.log("The hash of your transaction is: ", hash, "\nCheck Alchemy's Mempool to view the status of your transaction!"); 
+          } else {
+            console.log("Something went wrong when submitting your transaction:", err)
+          }
+        });
+      }).catch((err) => {
+        console.log(" Promise failed:", err);
+      });
+    }
+
+    mintNFT("https://gateway.pinata.cloud/ipfs/QmYueiuRNmL4MiA2GwtVMm6ZagknXnSpQnB3z2gWbz36hP")
+
+Now, run node scripts/mint-nft.js to deploy your NFT. After a couple of seconds, you should see a response like this in your terminal:
+
+    The hash of your transaction is: 0x10e5062309de0cd0be7edc92e8dbab191aa2791111c44274483fa766039e0e00
+
+    Check Alchemy's Mempool to view the status of your transaction!
+
+Next, visit your [Alchemy mempool](https://dashboard.alchemyapi.io/mempool) to see the status of your transaction (whether it’s pending, mined, or got dropped by the network). If your transaction got dropped, it’s also helpful to check [Ropsten Etherscan](https://ropsten.etherscan.io/) and search for your transaction hash.
+
+![View your NFT transaction hash on Etherscan](https://cdn-images-1.medium.com/max/5200/0*Q7IfLIhtVnNtAJn-.png)*View your NFT transaction hash on Etherscan*
+
+And that’s it! You’ve now deployed AND minted with a NFT on the Ethereum blockchain 🤑
```

Comment:
We use an `Emoji` component to standardize appearance of all Emoji's.. Just change this line to 
`And that’s it! You’ve now deployed AND minted with a NFT on the Ethereum blockchain <Emoji text=":money_mouth_face:" size={1} />`